src/pkg/backup/details/details.go

```diff
@@ -150,11 +162,30 @@ func FolderEntriesForPath(parent *path.Builder) []folderEntry {
 		})
 
 		parent = nextParent
+
+		if lfs != nil {
+			lfs = lfs.Dir()
+		}
 	}
 
 	return folders
 }
 
+// assumes the pb contains a path like:
+// <tenant>/<service>/<owner>/<category>/<logical_containers>...
+// and returns a string with only <logical_containers>/...
+func locationRefOf(pb *path.Builder) *path.Builder {
```

Comment:
we should probably plan to move this to some service/category specific code down the road. Depending on how we want the location refs to look it could vary between them

We should probably also refactor the code that adds folders to be more efficient, but that's also another PR